src/components/ChargeFormSections/ChargeInfoForm/ChargeInfoForm.js

```diff
@@ -108,7 +108,6 @@ const ChargeInfoForm = () => {
             name="exchangeRate.toCurrency"
             onChange={handleCurrencyChange}
             required
-            validate={requiredValidator}
```

Comment:
If this is required it needs a validator, if it isn't we should remove the `required` styling